### ISSUE TITLE
[cli] Copy password to clipboard instead of printing it in the console

### ISSUE THREAD

**Issue by oscarsaleta** [lesspass/lesspass#117]
I think it would be convenient if passwords using the cli app weren't printed to the console but instead copied to the clipboard.
I found this "copy-paste" package: https://github.com/xavi-/node-copy-paste, which allows to copy a string to clipboard.
Suggested modifications: `npm install --save --save-exact copy-paste` and modify `cli.js`:
```JS
'use strict';
var cpypst = require('copy-paste');
// ...
function calcPassword(site, login, masterPassword, options) {
    LessPass.generatePassword(site, login, masterPassword, options).then(generatedPassword => {
        cpypst.copy(generatedPassword);
        console.log("Copied to clipboard.");
        process.exit();
    });
}
```
(This solution works for me in Linux with *xclip* installed, I have not tested it anywhere else).

**Comment by guillaumevincent** [comment]
can we use a parameter to copy to clipboard instead of printing into the console ?
I use cli with other programs to generate passwords.

`--copy-to-clipboard` or `--copy` for example?

**Comment by oscarsaleta** [comment]
Of course, I had not thought about that. How do I go about it? Do I fork lesspass/cli and do a pull request there or do I simply post the code here?

**Comment by guillaumevincent** [comment]
available with lesspass-cli@2.1.0 thanks for your PR

**Comment by oscarsaleta** [comment]
Thanks to you for this awesome app
